CLAUDE.md

```diff
@@ -0,0 +1,89 @@
+# Claude Code Agent Guidelines
+
+This document provides guidance for you as the Claude Code agent on how to work with this project's context rules system.
+
+## Rule Location and Structure
+
+This project uses **Cursor's Context Rules** system located in `.cursor/rules/`. **ALWAYS** read and load the content of relevant rules from this directory into context when working on the codebase.
+
+### Discovering Available Rules
+
+Read all files in the `.cursor/rules/` directory to discover what rules are available. Each rule file uses the `.mdc` extension (Markdown with frontmatter).
```

Comment:
amazing, I hope this will read the rule to run formatting before making any commit!